Jenkinsfile_CNP

```diff
@@ -69,15 +69,14 @@ withPipeline(type, product, component) {
 
     env.IA_CASE_DOCUMENTS_API_URL = "http://ia-case-documents-api-aat.service.core-compute-aat.internal/"
     env.IA_CASE_NOTIFICATIONS_API_URL = "http://ia-case-notifications-api-aat.service.core-compute-aat.internal/"
+    env.IA_CASE_PAYMENTS_API_URL = "http://ia-case-payments-api-aat.service.core-compute-aat.internal/"
```

Comment:
do you really need this? I think Jenkins uses the TEST_URL to access this service.